samples/bluetooth/central_hr/src/main.c

```diff
@@ -82,7 +82,7 @@ static u8_t discover_func(struct bt_conn *conn,
 		subscribe_params.ccc_handle = attr->handle;
 
 		err = bt_gatt_subscribe(conn, &subscribe_params);
-		if (err && err != -EALREADY) {
+		if (err < 0 && err != -EALREADY) {
```

Comment:
Some apps out there are going to have to adapt to this. We should somehow make it visible